cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -408,6 +421,12 @@ func migrateCoreDNSCorefile(client clientset.Interface, cm *v1.ConfigMap, corefi
 	}, metav1.UpdateOptions{}); err != nil {
 		return errors.Wrap(err, "unable to update the CoreDNS ConfigMap")
 	}
+
+	// Point the CoreDNS deployment to the `Corefile-backup` data.
+	if err := setCorefile(client, "Corefile-backup"); err != nil {
+		return err
```

Comment:
If this errors, then the original coredns is left pointing at a migrated corefile, which could result in a coredns that wont start.  Is this a concern?  Will this ever error? Fix would be only add `Corefile-backup` in the above configmap update, and then update the `Corefile` value after calling `setCorefile(client, "Corefile-backup")`.  This causes an extra deployment change though.  Is this worth doing?